x-pack/platform/plugins/shared/actions/server/lib/action_executor.ts

```diff
@@ -5,32 +5,25 @@
  * 2.0.
  */
 
-import type { PublicMethodsOf } from '@kbn/utility-types';
```

Comment:
Looking at the code changes in this module, it's not clear why there are so many changes in the imports.  It looks like some things were just moved around?  This is distracting, but also makes it harder to do "blame" research.  Can we get this closer (or the same) as the original version of the module?

Reply:
I resolved the import/export sorting that was done by the local IDE, the PR should now be much cleaner!